File: Virtual LMS/Exp 2.c
// Write a C program to find the average of n elements in an array

#include<stdio.h>
int main()
{
    int a[100],n,sum=0,avg;
    scanf("%d",&n);
    for(int i=0; i<n; i++)
    {
        scanf("%d",&a[i]);
    }
    for(int i=0; i<n; i++)
    {
        sum+=a[i];
    }
    avg=sum/n;
    printf("%d",avg);
    return 0;
}


// Input: 5, 5, 10, 15, 20, 25

// Output: 15

File: Practice Python/Exp 2.py
# Write a  Python program to perform linear search and find the element in the list 

arr=[int(x) for x in input().split()]
n= int(input())
c=0
for i in range(len(arr)):
    if n==arr[i]:
        print(i+1)
        c=1
        break
if c==0:
    print("The Element is not found")


# Input: 10, 25, 45, 12, 75, 45        Output:  3

# Input: 10, 25, 45, 12, 75, 45, 77    Output:  The Element is not found

